sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1280,6 +1280,14 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ENABLE_APPROX_PERCENTILE = conf("spark.rapids.approxPercentileEnabled")
```

Comment:
The framework already generates a config for each operator based off of the name of the Spark operation.  You can have it disabled by default by adding something like

```
        .disabledByDefault("...")
```

in the meta for it.  I think in this case the config is `spark.rapids.sql.expression.ApproximatePercentile`  You could probably drop this config and all of the checks for it and replace it with the one that the framework automatically generates and checks.